Slider/Assets/Scripts/Sliders/Grid/AreaSTiles/MirageSTileManager.cs

```diff
@@ -60,25 +60,28 @@ public void DisableMirage(int islandId = -1)
     /// If no mirage tile is active on that position, that position does not
     /// appear in the dictionary.
     /// </summary>
-    public static Dictionary<Vector2Int, int> GetActiveMirageTileIdsByPosition()
+    public static Dictionary<Vector2Int, char> GetActiveMirageTileIdsByPosition()
     {
         List<GameObject> mirageTiles = _instance.mirageSTiles;
 
-        Dictionary<Vector2Int, int> tileIdToPosition = new();
+        Dictionary<Vector2Int, char> tileIdToPosition = new();
 
         for (int tileId = 0; tileId < mirageTiles.Count; tileId++)
         {
             GameObject mirageTile = mirageTiles[tileId];
             if (mirageTile.activeInHierarchy)
             {
                 Vector2Int mirageTileGridPosition = GridPositionFromWorldPosition(mirageTile.transform.position);
-                tileIdToPosition[mirageTileGridPosition] = tileId + 1;
+                //tileIdToPosition[mirageTileGridPosition] = tileId + 1;
+                tileIdToPosition[mirageTileGridPosition] = alphabet[tileId];
             }
         }
 
         return tileIdToPosition;
     }
 
+    private static string alphabet = "abcdefghijklmnopqrstuvwxyz";
```

Comment:
nits: 
should we use capital letters like we do for magitech past tiles?
you don't need the full alphabet
you can do int to char conversion w/o a map